src/EquipmentEditor.cpp

```diff
@@ -730,7 +737,9 @@ void EquipmentEditor::updateCheckboxRecord()
    {
       double bar = calcBatchSize();
       lineEdit_boilSize->setText(bar);
+      lineEdit_boilSize->setReadOnly(true);
    }
+   else lineEdit_boilSize->setReadOnly(false);
```

Comment:
Personally I disagree. A disabled EditText makes it look as if the value is not important in this context. To quote from a MSDN forum post.

"Use ReadOnly when you have data that you want the user to see and copy, but not modify. Use a disabled textbox, when the data you are displaying is not applicable in for the current state of a dialog or window."

However, I will change it if you insist.